pkg/kev/converter/kubernetes/transform.go

```diff
@@ -1660,20 +1660,16 @@ func (k *Kubernetes) updateKubernetesObjects(projectService ProjectService, obje
 	}
 
 	// @step add PVCs to objects
-	if pvcs != nil {
-		// Looping on the slice pvcs instead of `*objects = append(*objects, pvcs...)`
-		// because the type of objects and pvcs is different, but when doing append
-		// one element at a time it gets converted to runtime.Object for objects slice
-		for _, p := range pvcs {
-			*objects = append(*objects, p)
-		}
+	// Looping on the slice pvcs instead of `*objects = append(*objects, pvcs...)`
+	// because the type of objects and pvcs is different, but when doing append
+	// one element at a time it gets converted to runtime.Object for objects slice
+	for _, p := range pvcs {
+		*objects = append(*objects, p)
 	}
 
 	// @step add ConfigMaps to objects
-	if cms != nil {
```

Comment:
What changed to allow removing this check?

Reply:
same as the one before